scripts/deploy/actions/azstor-prereq.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: &name azstor-prereq
```

Comment:
I'm not familiar with this syntax. Does `&name` declare a variable `name` that contains the value `azstore-prereq` and the value accessed by `*name`?

Reply:
Oh neat! This is called a Tag in [YAML Syntax](https://docs.fileformat.com/programming/yaml/#syntax). It defines a reference to a node and the node's contents can be accessed with the `*` operation.

I learned something new today!